### ISSUE TITLE
Diablo IV is not recognized as a Battle.net Game

### ISSUE THREAD

**Issue by DaBigBadDawg** [bartok765/galaxy_blizzard_plugin#77]
Diablo 4 does not appear in the list of Battle.Net playable games. 
Tried editing the definitions.py file and adding in the game info under TITLE_ID and Blizzard games but this just crashes the plugin.
The TITLE ID XXXXXXX is replaced with the number from my battle.net profile for  the game. 
        XXXXXXX: RegionalGameInfo('fenris', True) 
        BlizzardGame('fenris', 'Diablo IV', 'FENRIS')

**Comment by Crolug** [comment]
Indeed, it is not. But I guess this plugin is not actively maintained anymore.

**Comment by VeryGooDiS** [comment]
Diablo 4 does not appear in the list of Battle.Net playable games. Still. In 2025....
